src/v2/Apps/ArtistSeries/Components/ArtistSeriesHeader.tsx

```diff
@@ -164,13 +160,43 @@ const ArtistSeriesHeaderSmall: React.FC<ArtistSeriesHeaderProps> = props => {
   )
 }
 
+const StyledGrid = styled(Grid)`
+  @media (max-width: ${theme.flexboxgrid.breakpoints.lg - 1}px) {
+    max-width: 100%;
+  }
+`
+
+export const HeaderImage = styled(Image)`
+  border-radius: 2px;
+
+  @media (max-width: ${theme.flexboxgrid.breakpoints.sm - 1}px) {
+    max-width: 180px;
+    max-height: 180px;
+    margin: auto;
+  }
+
+  @media (min-width: ${theme.flexboxgrid.breakpoints.sm}px) {
+    max-height: 400px;
+    max-width: 100%;
+  }
+`
+
+const AlignedCol = styled(Col)`
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
```

Comment:
I wonder if these could have simply been provided as [styled-system props](https://github.com/artsy/palette/blob/c67faa0e56aaeff14a35b7f5c2171d83380ad0d8/packages/palette/src/elements/Grid/Grid.tsx#L45-L56) right on the `Col` element?

Reply:
Oh cool! That also makes sense. I did it this way because I didn't realize I could use those with `Col`.